platform/android/java/lib/src/org/godotengine/godot/Godot.kt

```diff
@@ -304,6 +297,26 @@ class Godot(private val context: Context) : SensorEventListener {
 		}
 	}
 
+	/**
+	 * Goes into immersive mode.
+	 * For now, once you go immersive, you can't go back.
+	 * Call only from UI thread.
+	 */
+	fun goImmersive() {
```

Comment:
Let's rename the method to `enableImmersive()` and give it `internal` visibility.

Reply:
This cannot be both `internal` and also accessed from GodotEditor, so I'll do the latter.